### PR TITLE
fixing annotation recipe

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- The loqus db headers weren't added to the correct file

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: definitions/rd_dna_initiation_map.yaml [hunk 1]
```diff
@@ -77,13 +77,13 @@ CHAIN_ALL:
...
     - bcftools_norm
     - variant_annotation
-    - mt_annotation
     - CHAIN_RHOVIZ:
       - rhocall_viz
       - chromograph_rhoviz
     - CHAIN_UPD:
       - upd_ar
       - chromograph_upd
+    - mt_annotation
```
Comment (projectoriented) [inline definitions/rd_dna_initiation_map.yaml:86]: Are the steps of the pipeline initiated from this yaml file from top down? What's the effect of mt_annot being moved here?

Comment (jemten) [reply definitions/rd_dna_initiation_map.yaml:86]: Shouldn't be any difference in the output of mt_annotation. However, upd_ar takes the output from the upstream recipe on the main chain which used to be the mt annotated file. While rhocall_viz is hardcoded to take the outfiles from variant_annotation. mt_annotation will always use the outfiles from variant _annotation regardless

Comment (projectoriented) [reply definitions/rd_dna_initiation_map.yaml:86]: Ahh okay, got it. 👍

**Review by projectoriented — APPROVED** [review]

**Comment by jemten** [comment]
I'll merge this now. There has been so many dangling PR:s that I'm hard pressed to remember how everything fitted together 😅 . There's quite possibly gonna be a new one that tries to tie everything together

**Pull request merged**